.github/workflows/ci.yml

```diff
@@ -7,12 +7,21 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        PHP_VERSION: [ "8.0", "8.1" ]
+        ROBO_VERSION: [ "4.0", "5.0" ]
+        PHP_VERSION: [ "8.2", "8.3" ]
+        include:
+          - ROBO_VERSION: "4.0"
+            PHP_VERSION: "8.0"
+          - ROBO_VERSION: "4.0"
+            PHP_VERSION: "8.1"
```

Comment:
There is something wrong here, the PHP versions listed are 8.2 and 8.3 but you include lower versions in the matrix.
4.x supports min version php8.0 but the 5.x supports only php8.2 and up

Reply:
Indeed, Robo 4 and 5 supports different versions of PHP. What is configured here for the pipeline is:
- Execute PHP 8.2 and 8.3 for both Robo versions.
- For Robo 4.0 execute in adittion 8.0 and 8.1.
Example: https://github.com/AaronGilMartinez/robo-config/actions/runs/11018721272

Reply:
I see it points to the wrong pipeline